analyzers/python/integration_test.go

```diff
@@ -18,12 +16,11 @@ import (
 
 var pythonAnalyzerFixtureDir = filepath.Join(fixtures.Directory(), "python", "analyzer")
 
-func TestMain(m *testing.M) {
-	// flags are not parsed at this point. In order to have testing.Short() read actually provided values, this must be executed
-	flag.Parse()
+func TestPythonAnalysis(t *testing.T) {
 	if testing.Short() {
 		return
```

Comment:
Actually, here and below: I'd prefer `if testing.Short() { t.Skip() }` over `if testing.Short() { return }`.